client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -63,13 +64,15 @@ public SidecarClient(SidecarInstancesProvider instancesProvider,
                          RetryPolicy defaultRetryPolicy)
     {
         this.defaultRetryPolicy = defaultRetryPolicy;
-        ignoreConflictRetryPolicy = new IgnoreConflictRetryPolicy(sidecarClientConfig.maxRetries(),
-                                                                  sidecarClientConfig.retryDelayMillis(),
-                                                                  sidecarClientConfig.maxRetryDelayMillis());
-        baseBuilder = new RequestContext.Builder()
-                      .instanceSelectionPolicy(new RandomInstanceSelectionPolicy(instancesProvider))
-                      .retryPolicy(defaultRetryPolicy);
-        executor = requestExecutor;
+        this.ignoreConflictRetryPolicy = new IgnoreConflictRetryPolicy(sidecarClientConfig.maxRetries(),
+                                                                       sidecarClientConfig.retryDelayMillis(),
+                                                                       sidecarClientConfig.maxRetryDelayMillis());
+        this.oncePerInstanceRetryPolicy = new OncePerInstanceRetryPolicy(sidecarClientConfig.minimumHealthRetryDelay(),
+                                                                         sidecarClientConfig.maximumHealthRetryDelay());
+        this.baseBuilder = new RequestContext.Builder()
+                .instanceSelectionPolicy(new RandomInstanceSelectionPolicy(instancesProvider))
+                .retryPolicy(defaultRetryPolicy);
```

Comment:
The indentation looks wrong. 
I also feel that adding the `this.` to instance variable unnecessarily increases the number of line of changes.

Reply:
Removed optional `this.`es, changed indentation.